meeting-notes/59-06Aug2020.md

```diff
@@ -0,0 +1,75 @@
+- clement: it was very interesting. `idili` talk was quite interesting. Having a device with a daemon to share things. It would be nice to see it with hyper.
+- mauve: lot of fun, was there for the  full conference! Lots of energy drinks. Def worth it. Having the recording right away is super handy! Its nice to link right away and share. Would like to hear feedback about the workshop. Nice to see people and chat. It would be nice to have a hangout area like gather.town.
+- fb: couldn't make it, gonna watch the recording later.
+
+### Kevin's demo
+
+LIVE DEMO :movie_camera:
+user goal: user want to make a website
+- different approaches
+    - wordpress
+    - hugo/jekyll
+    - use a desktop app / frontpage style
+    - demo using beaker
+
+### agregore
+
+#### migrating to mac issues
+
+- fb: do you have any issue if you want to migrate? how to reach for questions? open an issue
+- mauve: open an issue! :) most of the test are on linux, also using travis. Currently debugging mac compile.
+- fb: also have a linux machine, gonna try to build it there.
+
+#### app icon issues on win
+
+- clement: app logo not showing. maybe you need to specify something specific for windows.
+- mauve: using electron builder. gonna check their docs again.
+- clement: maybe there are diff types of icons.
+- mauve: working on a new release: directory listing and better err codes + cleaning app fetch interface. For example, there is a bad connectivity problem in my city these days, while local networks works, we are not able to reach due to having issues accessing bootstrap nodes. Working on a _hacky_ fetch method to skip bootstraps, directly obtain IP address and connect to it. Still working on the fetch interface. Also gonna try the icon fix for the new release.
+
+#### ideas
+
+- fb: like the idea to directly connect with someone
+- mauve: there are like two strategies, dht and direct method. Would like to support both using different methods. connect to a specific key maybe. It could be useful on local network scenarios with manually bootstrapping us. More resilient approach. Other improvements: persisting non-peers into disk. save ip addresses, on exit, next time you load, you can connect to an old ip as a bootstrap.
+- fb: figure it out security concerns, if you don't want people connecting to you anymore.
+- mauve: hard problem. there are unique id using hypercore, we have a deny list using those. E.g.: I only want to connect to such and such people.
+- micah: related to agregore's goal, this is going to be like a shell for creating apps on top of it.
+- mauve: agregore should be something that you can install and build on top of it. It could be forked too for build more specific things. kind of like progressing web apps. with regards to mobile stuff, rad is working on gateway browser!
```

Comment:
```suggestion
- mauve: agregore should be something that you can install and build on top of it. It could be forked too for build more specific things. Apps can be kind of like progressive web apps, but without needing to set up service workers or anything since it's local first by default. with regards to mobile stuff, rad is working on gateway browser!
```